node/request.js

```diff
@@ -226,32 +228,50 @@ TChannelRequest.prototype.resend = function resend() {
         opts[key] = self.options[key];
     }
     opts.timeout = self.timeout - self.elapsed;
```

Comment:
this is now wrong whenever the onConnected starts to send it, since non-trivial time has elapsed; please:
- move the `opts` building into `onConnected`
- inline the RHS expression here in the setTImeout use inside the not connected timer set